src/py42/_internal/clients/alerts.py

```diff
@@ -68,12 +82,17 @@ def _get_alert_rules(
         page_num=None,
         page_size=None,
     ):
+        # This method exists separately from `get_page()` because
+        # of the tenant ID parameter - trying to avoid it and avoid duplicate calls
+        # to retrieve it.
+
+        # This API expects the first page to start with zero.
+        page_num = page_num - 1
```

Comment:
We should probably document that this is being done under the hood. If someone uses the now public method and knows the API starts at 0, their first request will be for page -1.

Reply:
I will put it in the moduel